client/flutter/lib/pages/home_page.dart

```diff
@@ -18,9 +18,12 @@ class HomePage extends StatelessWidget {
         child: SafeArea(
           child: CustomScrollView(slivers: [
             SliverAppBar(
-                expandedHeight: 110,
-                backgroundColor: Colors.white,
-                flexibleSpace: Image.asset("assets/WHO.jpg")),
+              expandedHeight: 110,
+              backgroundColor: Colors.white,
+              flexibleSpace: Image.asset("assets/WHO.jpg"),
+              floating: false,
+              snap: false,
```

Comment:
As both `snap` and `floating` its default values in `SliverAppBar` are `false`, this doesn't change anything at all?